desk/app/channels.hoon

```diff
@@ -474,8 +615,28 @@
       (toggle-post toggle.a-channels)
     ?:  ?=(%join -.a-channel.a-channels)
       ca-abet:(ca-join:ca-core [nest group.a-channel]:a-channels)
+    ::  upconvert emoji
+    ::
+    =?  a-channel.a-channels  ?=([%post %add-react *] a-channel.a-channels)
+      %_  a-channel.a-channels
+        q.c-post
+        =*  react  q.c-post.a-channel.a-channels
+        ?>  ?=(@t react)
```

Comment:
If it's an old action, you wouldn't need this assertion, right?

Also I'd patch up the indentation like so:

```suggestion
      %_  a-channel.a-channels
          q.c-post
        =*  react  q.c-post.a-channel.a-channels
```

Same thing for the block below.